core/templates/dev/head/pages/topics_and_skills_dashboard/SkillsListDirective.js

```diff
@@ -40,6 +40,15 @@ oppia.directive('skillsList', [
             'description', 'worked_examples_count', 'misconception_count'
           ];
 
+          $scope.highlightedIndex = -1;
```

Comment:
null is better than -1 as a sentinel value, semantically.

Reply:
Done.